src/main/java/de/muenchen/isi/infrastructure/entity/enums/lookup/StandVerfahren.java

```diff
@@ -104,7 +106,8 @@ public static List<StandVerfahren> getStandVerfahrenForWeiteresVerfahren() {
             STAEDTEBAULICHE_SANIERUNGSMASSNAHME,
             STAEDTEBAULICHE_ENTWICKLUNGSMASSNAHME,
             INFO_FEHLT,
-            FREIE_EINGABE
+            FREIE_EINGABE,
+            STANDORTABFRAGE
```

Comment:
Da nur die Abfrage vom Typ `Weiteres Verfahren` die Standortabfrage beinhaltet, ist das Durchschleifen der Auswahlmöglichkeiten an das Frontend analog wie im Branch der Berechnung zu machen.

https://github.com/it-at-m/isi-backend/pull/118/files#diff-29324c1a4d1afa8376efa43cba90e2be60ff7f29df087372f0425bd904e41d9dL23

https://github.com/it-at-m/isi-backend/pull/118/files#diff-4775912094bfc6c323e41c5de8c4f28c149ebbf2d11bc31fddfc8eaf1eb07df6L250

Ich denke am besten ist es, wenn dieser Branch auf dem Berechnungsbranch basiert oder die Reviewanmerkung eingearbeitet wird, wenn der Berechnungsbranch gemerged ist.

Reply:
Mein Fehler. Die Klasse ist etwas anders Aufgebaut als die Klasse für den Sobon-Orientierungswert. Die Standortabfrage wird für das Weitere Verfahren eh bereits durchgeschleift.